docker-compose.yaml

```diff
@@ -2,13 +2,45 @@
 # This is not for running your Misago site in docker
 version: "3.0"
 services:
+  celery:
+    build: .
+    command: celery -A tasks worker --loglevel=info
+    environment:
+      # Postgres
+      - POSTGRES_USER=misago
+      - POSTGRES_PASSWORD=misago
+      - POSTGRES_DB=misago
+      - POSTGRES_HOST=postgres
+      - POSTGRES_TEST_DB=misago_test
+      # Superuser
+      - SUPERUSER_USERNAME=Admin
+      - SUPERUSER_EMAIL=admin@example.com
+      - SUPERUSER_PASSWORD=password
+    ports:
+      # Map port 8000 in the container to port 8000 on the host
+      # This way we can access the forum through http://localhost:8000
+      - "${MISAGO_DEVSERVER_PORT:-8000}:8000"
+    depends_on:
+      - postgres
+    tty: true
+    volumes:
+      # Map in the entire project into the container
+      # This makes sure files in the container updates on the fly as we were working locally
+      - .:/srv/misago:Z
   postgres:
     image: postgres:10
     environment:
       - POSTGRES_USER=misago
       - POSTGRES_PASSWORD=misago
     ports:
       - '127.0.0.1:5432:5432'
+  redis:
+    image: redis:5
+    restart: unless-stopped
+    networks:
+      - misago
+    volumes:
```

Comment:
Please don't create Redis volume for development setup. Those keep data between docker instances, and make it tricky to reset and clear application state.